### PR TITLE
[DOC] add issue template

### PR DESCRIPTION
**Pull request by h-2**
supersedes #123

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  - SeqAn version: <!-- enter version here -->
+  - Operating system: <!-- enter output of `uname -a` here-->
+  - Compiler: <!-- enter output of `/path/to/compiler --version` here -->
+
+## Description
+
+### How to repeat the problem
+
+<!-- step to repeat the problem, code snippets... -->
```
Comment (rrahn) [inline ISSUE_TEMPLATE.md]:
```suggestion
<!-- step to repeat the problem, code snippets, compiler flags... -->
```
In general I think that steps to reproduce the error is already including how the code was compiled but maybe writing it explicitly makes it more clear?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Regarding feature requests:
+###########################
+
+For feature requests you can remove all of the above, but clearly indicate that this is a feature request in the name, e.g. "[feature-request] add support for Fast5 format".
+
+One of the following will happen to feature requests:
+
+  * We will "accept" it and someone will start working on it for the next sprint [the issue will be kept open].
+  * We will convert it to a card in our [long-term project planning](https://github.com/seqan/seqan3/projects), but close the issue as no-one can work on it immediately.
+  * The feature request is invalid or not in the scope of the next releases and will be closed.
```
Comment (rrahn) [inline ISSUE_TEMPLATE.md]: * We will "accept" it and will include it for the current or the next release.
  * We will "accept" it but freeze the feature as there are no resources available to work on it. You are also very welcome to make a PR for it. Please read the [CONTRIBUTING](./CONTRIBUTING.md) guidlines for more details.
  * The feature request is invalid and will be closed.

Comment (rrahn) [reply ISSUE_TEMPLATE.md]: I have a slight different workflow in mind. Maybe we should discuss this the next strategy meeting?

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  * Search the list of existing issues to see if someone has reported the issue already.
+  * Update to the latest master branch of SeqAn and see if the issue persists.
+
+If you already know how to fix the issue, you can directly create a pull-request instead of an issue. See the guide on [CONTRIBUTING](./CONTRIBUTING.md) for more details.
+-->
+
+## Platform
+
+  - SeqAn version: <!-- enter version here -->
```
Comment (rrahn) [inline ISSUE_TEMPLATE.md]:
```suggestion
  - [ ] SeqAn version: <!-- enter version here -->
```

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  - Operating system: <!-- enter output of `uname -a` here-->
```
Comment (rrahn) [inline ISSUE_TEMPLATE.md]:
```suggestion
  - [ ] Operating system: <!-- enter output of `uname -a` here-->
```

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  - Compiler: <!-- enter output of `/path/to/compiler --version` here -->
```
Comment (rrahn) [inline ISSUE_TEMPLATE.md]:
```suggestion
  - [ ] Compiler: <!-- enter output of `/path/to/compiler --version` here -->
```

File: ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+<!-- what happens -->
+
+
+<!--
+###########################
+Regarding feature requests:
+###########################
+
+For feature requests you can remove all of the above, but clearly indicate that this is a feature request in the name, e.g. "[feature-request] add support for Fast5 format".
```
[the remark this answers is not in the document]
Comment (h-2) [reply ISSUE_TEMPLATE.md]: HM, but it's not a PR or a commit. It's an issue...

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (h-2) [reply ISSUE_TEMPLATE.md]: ok

File: ISSUE_TEMPLATE.md (hunk 3 above)
Comment (h-2) [reply ISSUE_TEMPLATE.md]: Why? This is the issue text that we are going to see, what's the point of us being able to check/un-check these? [Note that users creating the issue don't see checkboxes, they see the verbatim text inside the edit box]

**Pull request merged**